tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -0,0 +1,1497 @@
+// TODO(jwexler): Do we need these deps?
+//goog.require('goog.crypt.base64');
+//goog.require('jspb.Map');
+
+import BytesList from 'goog:proto.tensorflow.BytesList'; // from //third_party/tensorflow/core:protos_all_jspb_proto
```

Comment:
Please use `goog.require()` in TensorBoard codebase. We don't like ES6 imports.

Reply:
will be doing this in future PR before this component is pulled into the TB binary build